pddlgym/spaces.py

```diff
@@ -89,6 +89,25 @@ def _compute_all_ground_literals(self, state):
                 all_ground_literals.add(lit)
         return all_ground_literals
 
+    def contains(self, x):
+        """Return boolean if x is a valid member of this space."""
+        if not isinstance(x, State):
```

Comment:
should we actually `assert isinstance(x, State)` instead, or raise a `ValueError` if not?

Reply:
I think we should allow contains() to return False instead. Since this is a public method, it could be used outside of the library and the desired result is probably not to throw an error. Plus, looking at a reference environment code ([Box space in gym](https://www.gymlibrary.dev/api/spaces/#box): a box in real space R^n), they also opt to return False instead of throwing an error:

``` 
def contains(self, x) -> bool:
        """Return boolean specifying if x is a valid member of this space."""
        if not isinstance(x, np.ndarray):
            logger.warn("Casting input x to numpy array.")
            try:
                x = np.asarray(x, dtype=self.dtype)
            except (ValueError, TypeError):
                return False

        return bool(
            np.can_cast(x.dtype, self.dtype)
            and x.shape == self.shape
            and np.all(x >= self.low)
            and np.all(x <= self.high)
        )
```

Reply:
Sure, sounds good.